oak_runtime/src/lib.rs

```diff
@@ -275,6 +288,13 @@ pub enum ReadStatus {
     Success(Message),
     NeedsCapacity(usize, usize),
 }
+/// Helper type to indicate whether retrieving a serialized label has succeeded or has failed with
+/// not enough capacity.
+#[derive(Debug)]
+pub enum LabelSerializationStatus {
```

Comment:
Maybe we should rename the existing enums to `ChannelReadStatus`, and call this `LabelReadStatus` instead.